__tests__/app.test.js

```diff
@@ -78,4 +78,40 @@ describe('GET /api', () => {
             expect(result).toBe(true)
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('GET: 200 sends an array with article of correct ID', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article[0]).toEqual(expect.objectContaining({
+                article_id: 2,
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                article_img_url: expect.any(String),
+                votes: expect.any(Number)
+              }))
+        })
```

Comment:
In this instance we are requesting a single article and we know what we expect it to look like so this can be the hardcoded article 2.

I'd also send back rows[0] from the model so that you're only receiving a single article here and not the array.